Cargo.toml

```diff
@@ -26,7 +26,7 @@ picky-asn1 = "0.3.0"
 picky-asn1-der = "0.2.4"
 picky-asn1-x509 = "0.6.1"
 serde = "1.0.123"
-sha2 = "0.9.5"
+sha2 = "0.9.9"
 log = "0.4.14"
 rcgen = { version = "0.8.14", features = ["pem"] }
```

Comment:
Is there a reason why rcgen is pinned to 0.8.14 specifically?

Reply:
Hey, we've not updated all our dependencies, only the ones that had some problems coming from `cargo-audit`, and this was the most recent version when Paul created the CSR generation subcommand. Would you need this bumped to the latest release?